bigquery/google/cloud/bigquery/table.py

```diff
@@ -473,8 +473,9 @@ def _build_resource(self):
             resource['schema'] = {
                 'fields': _build_schema_resource(self._schema)
             }
-        else:
-            raise ValueError("Set either 'view_query' or 'schema'.")
+        elif self.partitioning_type is None:
+            raise ValueError(
+                "Set either 'view_query' or 'schema' or 'partitioning_type'.")
```

Comment:
@tswast During local nox test, I found that `test_create_w_alternate_client` is creating a create table request with both a schema and a view query. This used to succeed because the `elif` was there and the schema is not copied to into the request. I have a few options: 

1. Add the schema to the expected value in the request (because the request will now have both the view query and schema)
1. "fix" the test so it doesn't create the create table request with a schema. 
1. Change the test to remove the view query and add the schema to the expected value instead

Since the point of this test seems to be independent of the content of the request, the simplest solution is to remove the schema from the input (option 2).